rele/client.py

```diff
@@ -126,6 +126,7 @@ def publish(self, topic, data, blocking=False, **attrs):
         :param topic: string topic to publish the data.
         :param data: dict with the content of the message.
         :param blocking: boolean, default False.
+        :param timeout: float, default 3.0.
```

Comment:
it does not exactly default to 3. Multiple things happen here:
A) The class defaults to 3
B) Global init initializes the global publisher using `PUBLISHER_TIMEOUT` but defaults to 3
C) This method, `publish`, defaults to None, in which case it falls back to the class default (A) ), which may be 3 but it may not

So what's the right description for this docstring?

Reply:
In this API, default is 3. I forgot the docs for the `rele.publish` shortcut which would have a different behaviour.

Reply:
After a review, this seems to be a bigger issue related to the arguments being called in the `publish` api not being defined. As a result, Im going to open up a separate issue and not block this PR.